paimon-core/src/main/resources/META-INF/NOTICE

```diff
@@ -5,4 +5,6 @@ This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
 
 This project bundles the following dependencies under the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
-- com.squareup.okhttp3:okhttp:4.12.0
\ No newline at end of file
+- com.squareup.okhttp3:okhttp:4.12.0
+- com.squareup.okio:okio-jvm:3.6.0
+- org.jetbrains.kotlin:kotlin-stdlib:1.8.21
```

Comment:
If we don't need add okio-jvm and kotlin-stdlib as dependency. Needn't update this file.

Reply:
It still bundles `okio-jvm` and `kotlin-stdlib` dependencies in this project, It's just that these two dependencies are introduced by `okhttp3`.